src/main/java/frc/robot/controls/ControlBindings.java

```diff
@@ -54,4 +54,42 @@ public Optional<Trigger> seedFieldCentric() {
     return Optional.empty();
   }
 
+  public interface InnerControlBindings {
+
+    /**
+     * Intakes the Coral from the reef.
+     * 
+     * @return
+     */
+    public static Optional<Trigger> intakeCoralCommand() {
+      return Optional.empty();
+    }
+
+    /**
+     * Eject the Coral from the arm and Game Pieace Manipulator.
+     * 
+     * @return
+     */
+    public static Optional<Trigger> ejectCoralCommand() {
+      return Optional.empty();
+    }
+
+    /**
+     * Make sure the the Coral is in the right postion and allowes the arm to hold it.
+     * 
+     * @return
+     */
+    public static Optional<Trigger> activeHoldCoralComammand() {
```

Comment:
Method names should be verbs. Should this be `holdCoral`?
This should not be static. Implementors need to override it